data/examples/simba.cfg

```diff
@@ -112,6 +112,8 @@ loglevel = INFO
 # set to null to disable logfile creation
 # leave empty to have default [timestamp].log
 logfile =
+# can set different log level for log file
+loglevel_file =
```

Comment:
```
##### LOGGING #####
# minimum log level. Used for console and file if not specified otherwise in "loglevel_file". 
# Allowed/useful: DEBUG, INFO, WARN, ERROR.  (default: INFO) 
# INFO includes INFO, WARN and ERROR but excludes DEBUG
loglevel = INFO
# log file name. Placed in output directory
# set to null to disable logfile creation
# leave empty to have default [timestamp].log
logfile =
# Optional: set different log level for log file
loglevel_file =
```